docs/fuzzing_binary-only_targets.md

```diff
@@ -198,12 +198,15 @@ afl-clang-fast's.
 
 ### RetroWrite
 
-If you have an x86/x86_64 binary that still has its symbols, is compiled with
-position independent code (PIC/PIE), and does not use most of the C++ features,
+RetroWrite is a static binary rewriter that can be combined with AFL++. 
+If you have an x86_64 binary that still has its symbols (i.e., not stripped binary), 
+is compiled with position independent code (PIC/PIE), and does not contain C++ exceptions,
 then the RetroWrite solution might be for you. It decompiles to ASM files which
 can then be instrumented with afl-gcc.
 
-It is at about 80-85% performance.
+Binaries that are statically instrumented for fuzzing using RetroWrite are identical
+in performance to compiler-instrumented binaries and outperform 
+the QEMU-based instrumentation by 4.5x.
```

Comment:
can you please change this paragraph to "are close in performance" (because it is less efficient). and please remove the "by 4.5x" (as this is too target dependant I have seen closer to x3 in my experiments.

Reply:
Where are the QEMU numbers from? I'm not sure this holds true for qemuafl (but happy to learn otherwise)

Reply:
> can you please change this paragraph to "are close in performance" (because it is less efficient). and please remove the "by 4.5x" (as this is too target dependant I have seen closer to x3 in my experiments.

Sure! I will.

Reply:
> Where are the QEMU numbers from? I'm not sure this holds true for qemuafl (but happy to learn otherwise)

Well, actually I got them from the original paper of Retrowrite (https://ieeexplore.ieee.org/abstract/document/9152762)

Reply:
Thanks for the clarification. Looks like they compared against the old skool AFL qemu, a lot of improvements have happened since then , performance and instrumentation wise.